tests/unit_tests/test_ladder.py

```diff
@@ -85,3 +85,24 @@ def test_inform_player(ladder_service: LadderService):
 
     print(history)
     assert history == [1]
+
+
+async def test_get_ladder_history_many_maps(ladder_service: LadderService, players, db_engine):
+    game_id_start = 41935  # Some arbitrary number
+    num_maps = 7
+    async with db_engine.acquire() as conn:
+        for i in range(num_maps):
+            await conn.execute(
+                f"""insert into game_stats
+                    (id, startTime, gameType, gameMod, host, mapId, gameName, validity)
+                    values ({game_id_start+i}, NOW() + interval {i+1} minute, '0', 6, 1, {i}, 'MapRepitition', 0)"""
+                )
+            await conn.execute(
+                f"""insert into game_player_stats
+                    (gameId, playerId, AI, faction, color, team, place, mean, deviation, scoreTime)
+                    values ({game_id_start+i}, 1, 0, 0, 0, 2, 0, 1500, 500, NOW() + interval {i+1} minute)
+                """)
```

Comment:
If you could add comments on these ones it would be great - they're kind of hard to read and i'm not sure of what they do